src/server.rs

```diff
@@ -22,13 +25,19 @@ pub async fn run(ipc: IPC<Event>, mut shutdown_signal: Receiver<()>) {
 }
 
 async fn try_run(
-    ipc: IPC<Event>,
+    ipc_event: IPC<Event>,
```

Comment:
Nit: I personally don't see a benefit of the added `_event` suffix (but am also not opposed to it)

Reply:
I had to do it because otherwise the program was throwing errors. However this is no longer needed due to other suggested changes.